src/technologies/c.json

```
@@ -1634,9 +1634,9 @@
     ],
     "description": "CivicTheme is an open source, inclusive and component-based design system. It was created so governments and corporations can rapidly assemble modern, consistent and compliant digital experiences.",
```

Dummy commit to trigger tests since PR was opened without providing a list of websites in the template format.

```suggestion
    "description": "CivicTheme is an open source, inclusive, and component-based design system. It was created so governments and corporations can rapidly assemble modern, consistent and compliant digital experiences.",
```